.gitignore

```diff
@@ -131,4 +131,8 @@ dist
 
 # build files
 dist
-types
\ No newline at end of file
+types
+
+# IDE
```

Comment:
These are not pushed to npm and help in sharing run config among team. I see no harm in keeping these. If you still want to remove them, I can approve.

Reply:
I would prefer not to have them, as stated above, we right now don't have any special run configuration!